tests/http/clients/__init__.py

```diff
@@ -0,0 +1,174 @@
+import abc
```

Comment:
These clients could be used also from other people to test their API, isn't? 
To me, they seem similar to those I had started to implement some time ago, but with extra features :) 
Do you think we can use these and make them available to the public?

Reply:
that's what I mentioned in the monthly meeting 😊 I want to unify all the clients, but I'll probably do that in another PR, not sure what I currently wrote is that good for other users :)

Reply:
Oh you are right, I already forgot already that we discussed in the meeting... 🤦‍♀️ 
Looks good so far :D